atd-toolbox/data_model/python/sql_templates/table_constraints.sql

```diff
@@ -1,5 +1,5 @@
 alter table public.units_cris add constraint unique_units_cris unique (crash_pk, unit_nbr);
 alter table public.units add constraint unique_units unique (crash_pk, unit_nbr);
 alter table public.people_cris add constraint unique_people_cris unique (unit_id, prsn_nbr);
-alter table public.people_edits add constraint unique_people_edits unique (unit_id, prsn_nbr);
-alter table public.people add constraint unique_people unique (unit_id, prsn_nbr);
+alter table public.people_edits add constraint unique_people_edits unique (unit_id, prsn_nbr) deferrable initially deferred;
+alter table public.people add constraint unique_people unique (unit_id, prsn_nbr) deferrable initially deferred;
```

Comment:
this allows us to swap `people.unit_id` values within an update transaction. i've tested this via Hasura and it works as expected:

```graphql
mutation {
  update_people_edits(
    where: { id: { _eq: 2 } }
    _set: { unit_id: 1 }
  ) {
    affected_rows
  }
  # you alias update_people_edits_v2 in order to re-use it in the same mutation
  update_people_edits_2: update_people_edits(
    where: { id: { _eq: 1 } }
    _set: { unit_id: 2 }
  ) {
    affected_rows
  }
}
```

Reply:
And just for reference, we have this exact situation with crash ID `18790889`. If you look at this crash locally, the fatality is assigned to the wrong unit nbr. we apparently manually corrected this in prod DB. in any case, it will need to be manually fixed in post-go live.